opencog/README.md

```diff
@@ -54,7 +54,7 @@ You can use docker-compose for configuring your workspace on linux and Mac syste
     * cat /etc/hosts   # take note of the ip address for relex, e.g.
       `172.17.0.69     relex 8e7dc3a09f12 opencog_relex_1`
     * rlwrap telnet localhost 17001
-    * (define relex-server-host "172.17.0.69")
+    * (relex-server-host! "172.17.0.69")
```

Comment:
BTW, it doesn't have to be a numeric IP address, it can be a regular hostname e.g `relex-server.oclabs.polyu.hk` or something like that.